internal/pkg/externalplugins/plugins_test.go

```diff
@@ -100,10 +94,6 @@ func TestStartLoadConfig(t *testing.T) {
 
 	// Wait a moment.
 	time.Sleep(pullDuration + 1)
-	if config.TiCommunityLgtm[expectLen-1].ReviewActsAsLgtm == false {
```

Comment:
I'm curious about this original test. It seems we don't ever update config.

Reply:
Hmmmm that's an invalid assertion. It didn't read the update.